framework/src/partitioner/PetscExternalPartitioner.C

```diff
@@ -93,6 +93,15 @@ PetscExternalPartitioner::partition(MeshBase & mesh, const unsigned int n_parts)
   if (mesh.is_replicated() && n_parts > 1)
     preLinearPartition(mesh);
 
+  if ((mesh.n_elem() / n_parts < 28) && _part_package == "parmetis")
+  {
+    Moose::out
+        << "Number of elements per compute node must be less than 28, otherwise we switch to "
+           "PTScotch"
```

Comment:
this is the opposite of what it's doing though

Reply:
```suggestion
        << "Number of elements per compute node must be more than 28, otherwise we switch to "
           "PTScotch"
```
Indeed. I didn't see that until you pointed it out